src/main/scala/mesosphere/marathon/core/health/Health.scala

```diff
@@ -6,6 +6,7 @@ import mesosphere.marathon.state.Timestamp
 
 case class Health(
     instanceId: Instance.Id,
+    // TODO: add taskId here
     firstSuccess: Option[Timestamp] = None,
```

Comment:
For both Health and HealthResult, I'm torn between living it as is (additional Task.Id where needed) and replacing every occurence of Instance.Id by Task.Id (Instance.Id can be computed from Task.Id). The latter seems more coherent but it will be a bigger patch with possible side effects.

Reply:
Fair enough